cobalt/devinfra/kokoro/bin/build.sh

```diff
@@ -9,7 +9,7 @@ WORKSPACE_COBALT="${KOKORO_ARTIFACTS_DIR}/git/src"
 cd "${WORKSPACE_COBALT}"
 
 # Clean up workspace on exit or error.
-trap "bash ${WORKSPACE_COBALT}/internal/kokoro/bin/cleanup.sh" EXIT INT TERM
+trap "bash ${WORKSPACE_COBALT}/cobalt/devinfra/kokoro/bin/cleanup.sh" EXIT INT TERM
```

Comment:
Does this need src/ prefix?

Reply:
Does line 8 above answer your question?
As far as I can tell GitHub's lousy pull-request UI
won't let me comment on that line directly.